src/LocalAgent/Dockerfile

```diff
@@ -1,28 +1,31 @@
 #See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
 
-FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
+FROM mcr.microsoft.com/dotnet/aspnet:6.0-cbl-mariner2.0 AS base
 WORKDIR /app
 EXPOSE 80
 EXPOSE 443
 
-FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
+FROM mcr.microsoft.com/dotnet/sdk:6.0-cbl-mariner2.0 AS build
 WORKDIR /src
-COPY ["nuget.config", "."]
-COPY ["LocalAgent/localagent.csproj", "LocalAgent/"]
-COPY ["library/library.csproj", "library/"]
-COPY ["common/common.csproj", "common/"]
-COPY ["common.json/common.json.csproj", "common.json/"]
-RUN dotnet restore "LocalAgent/localagent.csproj"
+
+COPY ["/src/nuget.config", "."]
+COPY ["/src/LocalAgent/LocalAgent.csproj", "LocalAgent/"]
+COPY ["/src/library/library.csproj", "library/"]
+COPY ["/src/common/common.csproj", "common/"]
+COPY ["/src/common.json/common.json.csproj", "common.json/"]
+
+
+RUN dotnet restore "LocalAgent/LocalAgent.csproj"
 COPY . .
 WORKDIR "/src/LocalAgent"
-RUN dotnet build "localagent.csproj" -c Release -o /app/build
+RUN dotnet build "LocalAgent.csproj" -c Release -o /app/build
 
 FROM build AS publish
-RUN dotnet publish "localagent.csproj" -c Release -o /app/publish
+RUN dotnet publish "LocalAgent.csproj" -c Release -o /app/publish
 
 FROM base AS final
-RUN apt-get update
-RUN apt-get install iptables
+RUN tdnf install -y \
```

Comment:
💡 I wonder if this is some auto tool reindenting and removing `update.`

Reply:
I think the purpose here is to use tndf package manager which comes with CBL-Mariner which might not need update any more. But Tom can always correct me if  iam wrong. Thank you. :)

Reply:
yeah, mariner doesn't require update and uses tdnf instead of apt :)

Reply:
replaced with tdnf update command :)